wiki/Do_you_really_want_to_ask_peppy/pt-br.md

```diff
@@ -0,0 +1,15 @@
+# Você realmente quer perguntar para o peppy?
+
+Olá! peppy entende seu desejo de contactá-lo pessoalmente, mas ele pede para que primeiro entenda que este **não** é o lugar para pedir ajuda ou suporte para ele. Ele mantém o suporte do osu! separado de suas mensagens pessoas e também acha que responder PMs no fórum é difícil de manter. Por esse motivo, ele pede que você utilize uma das seguintes opções:
+
+## Suporte técnico
+
+A maioria dos problemas técnicos podem ser resolvidos procurando ou postando no [Fórum de ajuda](https://osu.ppy.sh/community/forums/5). Existem várias pessoas lá prontos para te ajudar, assim você encontrará uma solução mais rápido do que contactando ele.
```

Comment:
```suggestion
A maioria dos problemas técnicos podem ser resolvidos procurando ou postando no [Fórum de ajuda](https://osu.ppy.sh/community/forums/5). Existem várias pessoas lá prontas para te ajudar, assim você encontrará uma solução mais rápido do que contatando ele.
```
Same thing here, `contatar` vs `contactar`. `prontas` because `Pessoas`